templates/core.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Red | CALDERA</title>
+        <title>{{ page_title }} team | CALDERA</title>
```

Comment:
This shows up as i.e. 'Red Team team'

Reply:
Ah, great catch!

Reply:
I just made an update, could you please take another look?